Forgot to update the sha when I changed the rev, haha

Updated the commits based on those changes, but now install doesn't work for me locally; it errors out with:

       > installing
       > install flags: SHELL=/nix/store/3j18grljsyy4nxc078g00sy4cx6cf16g-bash-5.1-p16/bin/bash PREFIX=\$\(out\) install
       > install -m644 warpd.1.gz /nix/store/js9s9sw7g0irhzn4i1a02gpglg29jxzy-warpd-1.3.2/share/man/man1/
       > install: cannot create regular file '/nix/store/js9s9sw7g0irhzn4i1a02gpglg29jxzy-warpd-1.3.2/share/man/man1/': No such file or directory
       > make: *** [Makefile:63: install] Error 1
       For full logs, run 'nix log /nix/store/hwrbbdn26bsrfpydbp6p2vslfmkjfdpc-warpd-1.3.2.drv'.

(This is installing through `nix-env -f ~/s/nixpkgs -iA warpd`)

Adding this allows it to install without a problem:

    preConfigure = ''
      mkdir -p $out/share/man/man1
      mkdir -p $out/bin
    '';

Which seems odd, since you had mentioned in an earlier suggestion that Nix should do this automatically during the install phase. This bit of code isn't included in the most recent commits for that reason but I should note that at this PR's current state I couldn't get the package to build.

--- a/maintainers/maintainer-list.nix
+++ b/maintainers/maintainer-list.nix
@@ -5107,6 +5107,12 @@
     githubId = 3656888;
     name = "hhm";
   };
+  hhydraa = {
+    email = "hcurfman@keemail.me";
+    github = "hhydraa";
+    githubId = 58676303;
+    name = "hhydraa";
+  };
   higebu = {
     name = "Yuya Kusakabe";
     email = "yuya.kusakabe@gmail.com";

--- a/pkgs/applications/misc/warpd/default.nix
+++ b/pkgs/applications/misc/warpd/default.nix
@@ -1,0 +1,61 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, git
+, libXi
+, libXinerama
+, libXft
+, libXfixes
+, libXtst
+, libX11
+, libXext
+, waylandSupport ? false, cairo, libxkbcommon, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "warpd";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "warpd";
+    rev = "v${version}";
+    sha256 = "AR/uLgNX1VLPEcfUd8cnplMiaoEJlUxQ55Fst62RnbI=";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [ git ];
+
+  buildInputs =  [
+    libXi
+    libXinerama
+    libXft
+    libXfixes
+    libXtst
+    libX11
+    libXext
+  ] ++ lib.optionals waylandSupport [
+    cairo
+    libxkbcommon
+    wayland
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace '-m644' '-Dm644' \
+      --replace '-m755' '-Dm755' \
+      --replace 'warpd.1.gz $(DESTDIR)' 'warpd.1.gz -t $(DESTDIR)' \
+      --replace 'bin/warpd $(DESTDIR)' 'bin/warpd -t $(DESTDIR)'
+  '';
+
+  meta = with lib; {
+    description = "A modal keyboard driven interface for mouse manipulation.";
+    homepage = "https://github.com/rvaiya/warpd";
+    changelog = "https://github.com/rvaiya/warpd/blob/${src.rev}/CHANGELOG.md";
+    maintainers = with maintainers; [ hhydraa ];
+    license = licenses.mit;
+    platforms = platforms.linux;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -30984,6 +30984,8 @@ with pkgs;
 
   warp = callPackage ../applications/networking/warp { };
 
+  warpd = callPackage ../applications/misc/warpd { };
+
   w3m = callPackage ../applications/networking/browsers/w3m { };
 
   # Should always be the version with the most features

